### PR TITLE
WIP: SOA Preview Guide

### PR DESCRIPTION
**Pull request by markxnelson**
@markxnelson and @rosemarymarano are using the PR to collaborate on the SOA Preview Guide - which will add and end-to-end walkthrough for using SOA on k8s with the operator.
Do not merge yet!

### CODE REVIEW

File: docs-source/content/samples/simple/soa-preview-guide/_index.md [hunk 1]
```diff
@@ -0,0 +1,1216 @@
...
+      - name: fix-pvc-owner
+        image:  busybox
+        command: ["sh", "-c", "chown -R 54321:54321 /ORCL"]
+        volumeMounts:
+        - name: soadb-storage
+          mountPath: /ORCL
+      # add the lines before this line
+      containers:
+      - name: soadb
+        image: coantiner-registry.oracle.com/database/enterprise:12.2.0.1
```
Comment (tbarnes-us) [inline docs-source/content/samples/simple/soa-preview-guide/_index.md]: type in image string

Comment (tbarnes-us) [reply docs-source/content/samples/simple/soa-preview-guide/_index.md]: typo in my comment too :-)

Comment (rosemarymarano) [reply docs-source/content/samples/simple/soa-preview-guide/_index.md]: Got it thanks (there were two of those).

**Comment by bsmoyers** [comment]
Can one of the admins verify this patch?

**Pull request closed without merging**